evpp/libevent.h

```diff
@@ -12,8 +12,6 @@
 
 #ifdef _WIN32
 #include <WinSock2.h>
-#else
-#include <sys/queue.h>
 #endif
```

Comment:
That's OK. We will be very appreciated if you can send a PR only included this change.